ruler-frontend/src/main/kotlin/com/spotify/ruler/frontend/components/Breakdown.kt

```diff
@@ -60,7 +60,11 @@ fun RBuilder.containerListItem(id: Int, container: FileContainer, sizeType: Meas
 @RFunction
 fun RBuilder.containerListItemHeader(id: Int, container: FileContainer, sizeType: Measurable.SizeType) {
     h2(classes = "accordion-header") {
-        button(classes = "accordion-button collapsed") {
+        var classes = "accordion-button collapsed"
+        if (container.files == null) {
+            classes = "$classes disabled"
+        }
+        button(classes = classes) {
```

Comment:
This has been simplified as suggested in [this comment](https://github.com/spotify/ruler/pull/102#discussion_r1028237072). I'm adding `disabled` class to the whole button so the view can't be expanded.